images/router/haproxy/conf/haproxy-config.template

```diff
@@ -8,6 +8,9 @@
 global
   # maxconn 4096
   daemon
+{{ with (env "ROUTER_SYSLOG_ADDRESS" "") }}
+  log {{.}} local1 {{env "ROUTER_LOG_LEVEL" "debug"}}
```

Comment:
Probably want the default level to be something other than `debug` like `warning`. Don't want it to be too chatty.